ui/src/pages/backoffice/Patron/PatronDetails/PatronDetails.js

```diff
@@ -93,6 +95,16 @@ export default class PatronDetails extends Component {
                           >
                             <PatronPendingLoans />
                           </Segment>
+                          <Header attached="top" as={'h3'}>
+                            Patron's current borrowing requests
```

Comment:
I would put more:
```suggestion
                            Patron's current interlibrary loans
```
WDYT?

Could you also please change `Patron's literature requests` -> `Patron's requests for new literatures`?